docs/pages/project/contributing/contributing-cli.md

```diff
@@ -128,6 +128,25 @@ When designing for the command line interface, ask and consider the following qu
 
 You will need to provide a short and long description of the command for the help pages and also for the Meshery Documentation.
 
+## Refactoring Mesheryctl Code
+
+The mesheryctl code needs to be refactored for the ease of management, to improve the readbility of the code, for making it easier for using the utility packages. 
+
+For now, the meshconfig has been refactored to have getters and setters which can further be used to manipulate any config. For example, reading or changing the context version has now been a lot easier with the help of specific getters and setters, a central function has also been added to write the config.yaml file.
```

Comment:
Refactor the historical viewpoint to that of present-state facts.

This comment applies to the entire update.